library/pom.xml

```diff
@@ -310,6 +310,11 @@
       <artifactId>commons-beanutils</artifactId>
       <version>1.8.3</version>
     </dependency>
+    <dependency>
+      <groupId>net.sf.trove4j</groupId>
+      <artifactId>trove4j</artifactId>
+      <version>3.0.3</version>
+    </dependency>
   </dependencies>
```

Comment:
@davidyan74 We know. This is part of a bigger change that Bright is making to move some classes to Malhar. The changes are being done incrementally and the removal of trove4j will be done as a separate change shortly.